crates/db_schema/src/diesel_ltree.patch

```diff
@@ -2,23 +2,6 @@ diff --git a/crates/db_schema/src/schema.rs b/crates/db_schema/src/schema.rs
 index 255c6422..f2ccf5e2 100644
 --- a/crates/db_schema/src/schema.rs
 +++ b/crates/db_schema/src/schema.rs
-@@ -2,16 +2,12 @@
```

Comment:
I had to remove this, because this PR adds the `PostViewModeEnum` in some conflicting lines.

This patch still retains the `diesel_ltree` below, and it did work locally.